app/views/settings.html

```diff
@@ -57,17 +57,112 @@
                 </div><!-- /resource details -->
 
 
-                <div ng-if="(quotas | hashSize) === 0">
+                <div ng-if="!(quotas | hashSize) && !(clusterQuotas | hashSize)">
                   <h2>
                     <span>Quota</span>
                   </h2>
                   <div class="help-block">{{quotaHelp}}</div>
                   <p><em>{{emptyMessageQuotas}}</em></p>
                 </div>
 
+                <div ng-repeat="quota in clusterQuotas | orderBy: 'metadata.name'" class="gutter-bottom">
+                  <h2>
+                    Cluster Quota <span ng-if="(clusterQuotas | hashSize) > 1">{{quota.metadata.name}}</span>
+                  </h2>
+                  <div ng-if="$first" class="help-block">Limits resource usage across a set of projects.</div>
+                  <dl ng-if="quota.spec.quota.scopes.length">
+                    <dt>Scopes:</dt>
+                    <dd>
+                      <div ng-repeat="scope in quota.spec.quota.scopes">
+                        {{scope | startCase}}
+                        <span class="text-muted small">
+                          <span ng-switch="scope" class="hide-ng-leave">
+                            <span ng-switch-when="Terminating">&mdash; Matches pods that have an active deadline.</span>
+                            <span ng-switch-when="NotTerminating">&mdash; Matches pods that do not have an active deadline.</span>
+                            <span ng-switch-when="BestEffort">&mdash; Matches pods that have best effort quality of service.</span>
+                            <span ng-switch-when="NotBestEffort">&mdash; Matches pods that do not have best effort quality of service.</span>
+                          </span>
+                        </span>
+                      </div>
+                    </dd>
+                  </dl>
```

Comment:
Is your concern mainly around the messages being duplicated? i'd rather add a filter then make this a directive i think